.clang-format

```
@@ -28,3 +28,4 @@ EmptyLineBeforeAccessModifier: Leave
 #PackConstructorInitializers: BinPack
 BreakBeforeBinaryOperators: NonAssignment
 AlwaysBreakBeforeMultilineStrings: true
+IndentPPDirectives: AfterHash
```

Maybe we should also consider setting `PPIndentWidth`: https://clang.llvm.org/docs/ClangFormatStyleOptions.html#ppindentwidth

```suggestion
IndentPPDirectives: AfterHash
PPIndentWidth: 1
```

---

I think two is better, I was doing 1 the first time to "get back on" even character offsets, but if it thinks it is better to do 1, 3, 5, etc. I am OK with that.

---

In that case, let's go with 2. I don't have a preference; I only suggested 1 because I saw you went with one on #10486

---

```suggestion
IndentPPDirectives: AfterHash
PPIndentWidth: 2
```
OK sounds good